### PR TITLE
HPCC-16531 Label filtered disk count/exist in the graph

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

### CODE REVIEW

**Comment by ghalliday** [comment]
@shamser please review

File: ecl/hqlcpp/hqlsource.cpp [hunk 1]
```diff
@@ -1872,15 +1888,15 @@ ABoundActivity * SourceBuilder::buildActivity(BuildCtx & ctx, IHqlExpression * e
 
         if ((translator.getTargetClusterType() == RoxieCluster) && isSpill)
             graphLabel.append("Read");
+        else if (isExplicitExists() && (activityKind == TAKdiskcount))
```
Comment (shamser) [inline ecl/hqlcpp/hqlsource.cpp]: Should it also check for activityKind==TAKdiskexists?

**Review by shamser — COMMENTED** [review]
@ghalliday Just one comment

File: ecl/hqlcpp/hqlsource.cpp (hunk 1 above)
Comment (ghalliday) [inline ecl/hqlcpp/hqlsource.cpp]: It doesn't need to since that would already output "Disk Exists"

**Comment by ghalliday** [comment]
@shamser see answer
@richardkchapman please review

**Pull request merged**